ironic-deployment/ironic/ironic.yaml

```diff
@@ -90,12 +90,34 @@ spec:
                 secretKeyRef:
                   name: mariadb-password
                   key: password
+        - name: ironic-log-watch
+          image: quay.io/metal3-io/ironic
+          imagePullPolicy: Always
+          command:
+            - /bin/runlogwatch.sh
+          volumeMounts:
+            - mountPath: /shared
+              name: ironic-data-volume
+          envFrom:
+            - configMapRef:
+                name: ironic-bmo-configmap
```

Comment:
Does the script need the configuration data from the ConfigMap?

Reply:
I don't think so. To be removed.